charts/backstage/Chart.yaml

```diff
@@ -26,22 +30,24 @@ dependencies:
 keywords:
   - backstage
   - idp
-  - janus-idp
-kubeVersion: ">= 1.25.0-0"
+  - developer-hub
+  - redhat-developer-hub
+  - redhat
+kubeVersion: ">= 1.27.0-0"
 maintainers:
-  - name: Red Hat Developer Hub Team
-    url: https://github.com/redhat-developer/rhdh-chart
+  - name: Red Hat
+    url: https://redhat.com
 name: backstage
```

Comment:
this is being changed to `redhat-developer-hub` downstream in https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/helm/prepare.sh#L329-332 so we can keep it  as backstage here.